### PR TITLE
Lucky kat

### PR DESCRIPTION
**Pull request by kat-sauma**

### CODE REVIEW

File: get-random-throw.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (brookeperkins) [inline get-random-throw.js:1]: remove this whitespace! make sure line 1 of your file always has something on it. you did the right thing convention-wise by leaving a line of whitespace at the end of this file. i do not know why this is the rule, but it is. 🙃

File: index.html [hunk 2]
```diff
@@ -14,15 +15,56 @@
...
+                <span id="total"> Total: </span>
+            </div>
+        </section>
     </main>
     <footer>
-        <div class="copy gutter-right">&copy; 2020 Alchemy Code Lab</div>
+        <div class="copy gutter-right">&copy; 2021 Lucky Kat Code</div>
     </footer>
 
     <script type="module" src="app.js"></script>
```
Comment (brookeperkins) [inline index.html:70]: add a line of whitespace at the bottom to follow convention!

File: get-random-throw.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+export function getRandomThrow(rockPaperScissor) {
+    if (rockPaperScissor === 1) {
+        return 'rock';
+    } else if (rockPaperScissor === 2) {
+        return 'paper';
+    } else {
+        return 'scissors';
+    }
+}
+
```
Comment (brookeperkins) [inline get-random-throw.js:11]: remove this whitespace (you'll only ever need one line of whitespace, never two).

**Comment by kat-sauma** [comment]
merging to collaborate

**Pull request merged**